scripts/mark-definite-framework-results.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+"""
+Uses a number of supplied json schemas to determine whether suppliers on a particular framework should be marked as
+onFramework. Attemots to recover a less strict subset of the schema from inside the declaration_definite_pass_schema
+at json path definitions/notDefiniteFail. This is used to differentiate between suppliers that definitely fail and those
+that will require a human decision ("discretionary"). If not found, all suppliers failing
+declaration_definite_pass_schema will have onFramework left unmodified (probably remaining null).
+
+Will also mark draft services with their respectivee validity (in the status field) as determined by
+draft_service_schema.
+
+Usage: mark-definite-framework-results.py [options] <stage> <api_token> <framework_slug>
+            <declaration_definite_pass_schema_path> [<draft_service_schema_path>]
+
+--updated-by=<user_string>        Specify updated_by string to use in API requests
+--dry-run                         Don't actually perform any updates on API
+--reassess-passed-sf              Don't skip supplier_frameworks with onFramework already True
+--reassess-failed-sf              Don't skip supplier_frameworks with onFramework already False
+--reassess-failed-draft-services  Don't skip draft_services with "failed" status
+-v, --verbose                     Produce more detailed console output
```

Comment:
Yes I was getting this too - I disregarded it and assumed it was part of the weird way we set up our logger...?